models/src/node/mod.rs

```diff
@@ -196,9 +196,9 @@ impl BottlerocketShadowSpec {
 #[derive(Clone, Debug, Default, Serialize, Deserialize, Eq, PartialEq, JsonSchema)]
 pub struct BottlerocketShadowStatus {
     #[validate(regex = "SEMVER_RE")]
-    current_version: String,
+    pub current_version: String,
```

Comment:
It might be better to keep these versions private and create a method to read these values since your monitor only tried to fetch the value. The reason we want it to be private is we don't want any method accidentally update these values which could cause issue for the behavior of agent and controller.

Reply:
One nice thing about Rust is that values are immutable by default, unless you use the `mut` keyword. I think for that reason interfaces can be a bit looser with making fields public. Since we have the code around setting this using a timestamp though, I agree that using a method is probably best.